### PR TITLE
Generate packageLocale element in bar descriptor

### PR DESCRIPTION
**Pull request by EricLeiLi**
packageLocale is required to pass build and release, thus we need to generate it based on config.xml;
it is related to "defaultlocale" attribute of "widget" node and language attribute of name and description nodes.

Fixes: https://github.com/blackberry/BB10-Webworks-Packager/issues/210

### CODE REVIEW

**Comment by jamesjhedley** [comment]
Needs rebase.

File: Framework [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 7b694a0319056d1c4751a98c6cd399b7fadf7fe7
+Subproject commit f4804752e9e294266ce7534245772df77d1c5503
```
Comment (jamesjhedley) [inline Framework:1]: If you're going to update the Framework hash, it should be the latest on next.

**Comment by jamesjhedley** [comment]
You have copied all the changes i made in my other pull request for localization into this one. Why?

**Comment by jamesjhedley** [comment]
I see now; You cherry-picked my change into your branch. This won't cause merge conflicts will it?

**Comment by EricLeiLi** [comment]
The commit is included in https://github.com/blackberry-webworks/BB10-Webworks-Packager/pull/108
because it depends on that pull request.

**Pull request closed without merging**